.github/workflows/export-audit-data-scheduler.yml

```
@@ -2,10 +2,8 @@
 name: Export Audit Data to CSV
```

Lets rename this to make the jobs a little easier to manage

```suggestion
name: Export Audit Data to CSV Scheduler
```